Tests/fixtures/TestReporting/testReport.html

```diff
@@ -1635,59 +944,21 @@ <h1>51</h1>
         <circle cx="43.063" cy="39.2" r="4.188" stroke-miterlimit="10"/>
         <path stroke-linecap="round" stroke-linejoin="round" d="M53.015 20.665a460.015 460.015 0 012.416-2.39s1.943 1.84 3.27 1.443c1.373-.41 2.156-1.808 2.035-2.954-.167-1.588-2.11-3.04-4.438-2.556.406-1.513.068-3.062-.933-3.995-1.654-1.54-4.078-.479-4.99.918-1.22 1.869 1.574 3.663 1.574 3.663a410.45 410.45 0 00-2.177 2.151M19.678 46.908c-1.549 1.54-2.6 2.58-2.92 2.89-.655-1.302-2.132-1.943-3.458-1.546-1.374.411-2.157 1.808-2.036 2.954.168 1.589 2.11 3.04 4.438 2.556-.405 1.513-.068 3.062.933 3.995 1.654 1.54 4.33.614 4.99-.918.47-1.088.193-2.297-1.438-3.505.26-.252 1.005-.99 2.11-2.087M19.113 20.792a489.893 489.893 0 00-2.543-2.516s-1.944 1.839-3.27 1.442c-1.374-.411-2.157-1.808-2.036-2.954.168-1.589 2.11-3.04 4.438-2.556-.405-1.513-.068-3.062.933-3.995 1.654-1.54 4.078-.479 4.99.918 1.22 1.869-1.573 3.663-1.573 3.663.286.277 1.157 1.14 2.447 2.421M52.497 47.082a535.186 535.186 0 002.744 2.716c.655-1.302 2.133-1.943 3.459-1.546 1.374.411 2.157 1.808 2.036 2.954-.168 1.589-2.11 3.04-4.438 2.556.405 1.513.068 3.062-.933 3.995-1.654 1.54-4.33.614-4.99-.918-.47-1.088-.193-2.297 1.438-3.505-.258-.25-.991-.975-2.078-2.055"/>
     </g>
-</svg></td>
-</tr>
-<tr>
-    <td class="left-aligned">file39.swift:39</td>
-    <td class="left-aligned"><wbr>Relational Operator Replacement<wbr></td>
-    <td class="mutation-snapshot"><div class="snapshot-changes">
-    <span class="snapshot-before">before</span>
-<span class="snapshot-arrow">→</span>
-<span class="snapshot-after">after</span>
-</div></td>
-    <td><svg class="passed" role="img" viewBox="0 0 72 72">
+</svg></td></tr><tr><td class="left-aligned">file39.swift:39</td><td class="left-aligned"><wbr>Relational Operator Replacement<wbr></td><td class="mutation-snapshot"><div class="snapshot-changes"><span class="snapshot-before">before</span><span class="snapshot-arrow">→</span><span class="snapshot-after">after</span></div></td><td><svg class="passed" role="img" viewBox="0 0 72 72">
```

Comment:
(randomly commenting on this line) does Plot have a "pretty-print" option that will do better line wrapping?

Reply:
I wasn't able to find anything other than indenting spaces 😔